src/utils/analytics/events.ts

```diff
@@ -0,0 +1,18 @@
+export enum HotspotEvents {
+  ADD_GATEWAY_INITIATED = 'AddGateway Initiated',
```

Comment:
snake_case would be more consistent with how we are doing tags in other contexts, like the DiagnosticsReport.

Reply:
I followed the [naming convention](https://segment.com/docs/getting-started/03-planning-full-install/#create-naming-conventions) from the Segment documentation. However, after checking the [DiagnosticsReport](https://github.com/NebraLtd/hm-pyhelper/blob/2409987b12e904ffe30c80ca5556096311043fec/hm_pyhelper/tests/test_diagnostic_report.py), it would be better to use snake_case names for our use case.